memote/support/helpers.py

```diff
@@ -173,8 +208,32 @@ def find_biomass_reaction(model):
     model : cobra.Model
         The metabolic model under investigation.
 
+    Returns
+    -------
+    list of biomass reactions
+
     """
-    return [rxn for rxn in model.reactions if "biomass" in rxn.id.lower()]
+    buzzwords = ['biomass', 'growth', 'bof']
+
+    buzzword_matches = set([rxn for rxn in model.reactions if any(
+        string in rxn.id.lower() for string in buzzwords
+    )])
+
+    sbo_matches = set([rxn for rxn in model.reactions if
```

Comment:
I would have checked for SBO presence first and only perform the other queries if that didn't yield results.